repositories/strategies/routingStrategy.js

```diff
@@ -67,12 +74,28 @@ const getRoutingRuleSetById = (trx, routingRuleSetId) => {
     .first();
 };
 
-const insertRoutingCondition = (trx, routingCondition) =>
-  trx("Routing_Conditions")
+const createSpecificConditionValue = async (trx, conditionId) =>
+  trx("Routing_ConditionValues")
+    .insert({
+      conditionId,
+      customNumber: null
+    })
+    .returning("*")
+    .then(head);
+
+const insertRoutingCondition = async (trx, routingCondition, answer) => {
+  const condition = await trx("Routing_Conditions")
     .insert(routingCondition)
     .returning("*")
     .then(head);
 
+  if (!isNil(answer) && includes(["Currency", "Number"], answer.type)) {
```

Comment:
Although, we don't currently have this and it's beyond the scope of this task.  I think at some point in the future we should make the answer types constants.